src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -191,11 +194,6 @@ remove all whitespace including the final carriage return.
 For the `-key` flag use the key created earlier and stored in the media.key
 file. However, when entering it on the command line, be sure you've removed its
 whitespace. For the `-key_id` flag repeat the key value.
```

Comment:
For example, DASH MPD(cenc:default_KID, see https://github.com/google/shaka-packager/blob/master/packager/app/test/testdata/bear-640x360-av-cenc-golden.mpd), fmp4 (in tenc box).